docs/terminology.mdx

```diff
@@ -101,9 +144,15 @@ The full details of the request object aren't that important or helpful to know,
       "description": "We use your social security number for identity verification and tax reporting."
     },
     {
-      "type": "LastNameCredential",
+      "type": "FullNameCredential",
```

Comment:
Should we mention the LastName here too?

Reply:
I don't think so

Reply:
https://github.com/UnumID/UnumID.github.io/blob/089a63370400869adddd6d3636a537015160edbb/docs/terminology.mdx#L127